pages/common/nmap.md

```diff
@@ -1,37 +1,37 @@
 # nmap
 
 > Network exploration tool and security/port scanner.
-> Some features only activate when Nmap is run with root privileges.
-> More information: <https://nmap.org>.
+> Some features like SYN scan activates only when `nmap` is run with root privileges.
+> More information: <https://nmap.org/book/man.html>.
 
-- Check if an IP address is up, and guess the remote host's operating system:
+- Scan (the top 1000) ports of a remote host with various [v]erbosity levels:
 
-`nmap -O {{ip_or_hostname}}`
+`nmap -v{{1|2|3}} {{ip_or_hostname}}`
 
-- Try to determine whether the specified hosts are up (ping scan) and what their names and MAC addresses are:
+- Run a ping sweep over an entire subnet or individual hosts very aggressively:
 
-`sudo nmap -sn {{ip_or_hostname}} {{optional_another_address}}`
+`nmap -T5 -sn {{192.168.0.0/24|ip_or_hostname1,ip_or_hostname2...}}`
 
-- Also enable scripts, service detection, OS fingerprinting and traceroute:
+- Enable OS detection, version detection, script scanning, and traceroute:
 
-`nmap -A {{address_or_addresses}}`
+`sudo nmap -A {{ip_or_hostname1,ip_or_hostname2...}}`
 
-- Scan a specific list of ports (use '-p-' for all ports from 1 to 65535):
+- Scan a specific list of ports (use `-p-` for all ports from 1 to 65535):
 
-`nmap -p {{port1,port2,...,portN}} {{address_or_addresses}}`
+`nmap -p {{port1,port2...}} {{ip_or_host1,ip_or_host2...}}`
 
-- Perform service and version detection of the top 1000 ports using default NSE scripts; writing results ('-oN') to output file:
+- Perform service and version detection of the top 1000 ports using default NSE scripts; writing results (`-oA`) to output files:
```

Comment:
Is there any file other than what's specified here (`top-100-ports`)?
```suggestion
- Perform service and version detection of the top 1000 ports using default NSE scripts; writing results (`-oA`) to the output file:
```

Reply:
@acuteenvy Yes, `-oA` produces 3 output files of different formats (`top-1000-ports.xml`, `top-1000-ports.nmap` and `top-1000-ports.gnmap`) and hence the word "files".